src/components/common/ThreeToggle/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { useTheme } from 'next-themes';
+import { useEffect, useId, useState } from 'react';
+import MonitorIcon from '../../../../public/assets/icons/monitor.svg';
+import MoonIcon from '../../../../public/assets/icons/moon.svg';
+import SunIcon from '../../../../public/assets/icons/sun.svg';
+import styles from './style.module.scss';
+
+const ThreeToggle = () => {
+  const { theme, setTheme } = useTheme();
+  const [mounted, setMounted] = useState(false);
+
+  const lightId = `light${useId()}`;
+  const systemId = `system${useId()}`;
+  const darkId = `light${useId()}`;
+
+  const iconSize = 30;
+
+  const calcIndicatorStyle = (newTheme: string | undefined) => {
+    switch (newTheme) {
+      case 'light':
+        return styles.switchOne;
+      case 'system':
+        return styles.switchTwo;
+      case 'dark':
+        return styles.switchThree;
+      default:
+        return styles.switchOne;
+    }
+  };
+
+  // IF YOU WANT TO ADD BACKGROUND TO SWITCH, USE THIS FUNCTION
+  // OR ADD VARS TO STYLES.SCSS AND VAR.SCSS
+  // const calcSwitchBackgroundStyle = (newTheme: string | undefined) => {
+  //   switch (newTheme) {
+  //     case 'light':
+  //       return styles.switchLightMode;
+  //     case 'system':
+  //       return systemTheme === 'light' ? styles.switchLightMode : styles.switchDarkMode;
+  //     case 'dark':
+  //       return styles.switchDarkMode;
+  //     default:
+  //       return styles.switchDarkMode;
+  //   }
+  // };
+
+  const calcAltText = (theme: string | undefined) => {
```

Comment:
```suggestion
  const altText = `Icon representing ${theme} theme is on`
```
Based on how you're using the theme variable, it looks like you could rewrite this entire switch as a single variable with a template literal

Reply:
How do I check/test for alt text?